frontend/fingertips-frontend/components/forms/IndicatorSelectionForm/index.tsx

```diff
@@ -138,14 +152,18 @@ export function IndicatorSelectionForm({
       />
       {searchResults.length ? (
         <>
-          <Checkbox
-            key={`select-all-indicator-${areAllIndicatorsSelected}`}
-            data-testid="select-all-checkbox"
-            defaultChecked={areAllIndicatorsSelected}
-            onChange={(e) => handleSelectAll(e.target.checked)}
-          >
-            Select all
-          </Checkbox>
+          <ResultLabelsContainer>
+            <Checkbox
+              key={`select-all-indicator-${areAllIndicatorsSelected}`}
+              data-testid="select-all-checkbox"
+              defaultChecked={areAllIndicatorsSelected}
+              onChange={(e) => handleSelectAll(e.target.checked)}
+            >
+              Select all
+            </Checkbox>
+
+            {showTrends ? 'Recent trend for selected area' : null}
```

Comment:
What we can do here is use the `Paragraph` component for this. That would remove the need for certain Font stying in the `ResultLabelsContainer` that we should then be getting for free.